vignettes/bisg.Rmd

```diff
@@ -3,53 +3,80 @@ title: "Bayesian Improved Surname Geocoding (BISG)"
 author: "null"
 date: "8/5/2020"
 output:
-  pdf_document: default
   html_document: default
+  pdf_document: default
 ---
 This vignette demonstrates how to perform Bayesian Improved Surname Geocoding when the race/ethncity of individuals are unknown within a dataset.
 
 ## *What is Bayesian Improved Surname Geocoding?*
 
 Bayesian Improved Surname Geocoding (BISG) is a method that applies the Bayes Rule/Theorem to predict the race/ethnicity of an individual using the individual's surname and geocoded location [Elliott et. al 2008, Elliot et al. 2009, Imai and Khanna 2016]. 
```

Comment:
"applies Bayes' Rule to predict..."

Reply:
noted change. I will make this sentence more concise